packages/twenty-server/src/database/commands/upgrade-version/0-40/0-40-phone-calling-code.command.ts

```diff
@@ -0,0 +1,344 @@
+    // Note: ask Felix if tit's ok for this breaking change TEXT to TEXT. At the same time, should we tranform countrycode en ENUM postgres & graphql ? answer : no need to worry about this breaking change.
+    // 4 - [IMO, not necessary] if we think it's important, update all additioanl phones to add a country code following the same logic
+
+    console.log(
+      `   Note : other consequesnces to check : zapier @martin + REST api @martin ✅
+          Note : other consequesnces to check : timeline activities @coco : ✅
+          Note : test with cutom object (not cutom field)
+    `,
+    );
+    // ---------------------------------FieldMetada-----------------------------------------------------------
+    // 1 - Add the calling code prop in the field-metadata defaultvalue
+    // 1bis - cahnge country code prop in the field-metadata defaultvalue cf above +33 => FR
+    // Note: no additionalPhgones in the default value
+
+    // ---------------------------------Seeds-----------------------------------------------------------
+    // 1 - Adjust seeders : update getDevSeedCompanyCustomFields and seedPeople
+
+    this.logger.log(`Part 1 - Workspace`);
+
+    // ------------------------------------------------------------------------------------------------------------------
+    // ------------------------------------------------------------------------------------------------------------------
+    let workspaceIterator = 1;
+
+    for (const workspaceId of workspaceIds) {
+      this.logger.log(
+        `Running command for workspace ${workspaceId} ${workspaceIterator}/${workspaceIds.length}`,
+      );
+
+      this.logger.log(
+        `P1 Step 1 - let's find all the fieldsMetadata that have the PHONES type, and extract the objectMetadataId`,
+      );
+
+      try {
+        const phonesFieldMetadata = await this.fieldMetadataRepository.find({
+          where: {
+            workspaceId,
+            type: FieldMetadataType.PHONES,
+          },
+        });
+
+        for (const phoneFieldMetadata of phonesFieldMetadata) {
+          if (
+            isDefined(phoneFieldMetadata) &&
+            isDefined(phoneFieldMetadata.name)
+          ) {
+            const [objectMetadata] = await this.objectMetadataRepository.find({
+              where: {
+                id: phoneFieldMetadata?.objectMetadataId,
+              },
+            });
+
+            this.logger.log(
+              `P1 Step 1 - Let's find the "nameSingular" of this objectMetadata: ${objectMetadata.nameSingular || 'not found'}`,
+            );
+
+            if (!objectMetadata || !objectMetadata.nameSingular) continue;
+
+            this.logger.log(
+              `P1 Step 1 - Create migration for field ${phoneFieldMetadata.name}`,
+            );
+
+            await this.workspaceMigrationService.createCustomMigration(
```

Comment:
You probably didn't mean to keep the field/column creation code in this command?

Reply:
I thought about deleting it. but came back from it. Since it is idempotent, and that we better avoid missing column creation rather than twicing the command, I left it as such.

Unless you have a strong opinion, I would suggest leaving it here.